### DISCUSSION TITLE
Monthly HNT

### DISCUSSION THREAD

**Discussion by paparick** [davetapley/helium-tax#67] [Ideas]
Love the tool! What about filtering for HNT earnings on a monthly basis?

**Comment by davetapley** [comment]
Do you mean on the download dropdown?

<img width="528" alt="helium-tax" src="https://user-images.githubusercontent.com/48232/152180762-2a967245-47e6-4bcb-b5d8-706dbfb9abfe.png">

**Comment by paparick** [comment]
Yes.

Rick Kamp
________________________________

**Reply by davetapley** [reply]
Because it takes a long time, or because you only need one month worth of data?
If the latter then I'd probably suggest filtering it in a spreadsheet application instead.
